docs/user-guide/systemrequirements.md

```diff
@@ -69,8 +71,10 @@ For more information on the relationship between the Zowe server components and
 
 The Zowe Application Framework server provides the Zowe Desktop that contains an extensible GUI with a number of applications allowing access to z/OS functions, such as the File Editor, TN3270 emulator, JES Explorer, and more. For more information, see [Zowe Architecture](../getting-started/zowe-architecture.md#zlux).
 
-- Node.js v8.x (except v8.16.1), and v12.x.
-  **Note:** when using v12.x, it is highly recommended that plug-ins used are tagged. For more information, see [Tagging on z/OS](../extend/extend-desktop/mvd-buildingplugins.md#tagging-plugin-files-on-z-os)
+- Node.js v8.x (except v8.16.1), v12.x, or v14
+  
+  **Note:** When using Node.js v12.x, it is highly recommended that plug-ins used are tagged. For more information, see [Tagging on z/OS](../extend/extend-desktop/mvd-buildingplugins.md#tagging-plugin-files-on-z-os). 
```

Comment:
tagging will apply to every version v12 and afterward, such as v14 and whatever version comes after v14.